.eslintignore

```diff
@@ -0,0 +1,2 @@
+src/commons/editor/Editor.tsx
+src/commons/sideContent/SideContentSubstVisualizer.tsx
```

Comment:
Maybe just

```tsx
/* eslint-disable simple-import-sort/sort */
```

at the top of these files instead of disabling ESLint entirely on those files?